.eslintrc.js

```diff
@@ -180,6 +184,8 @@ module.exports = {
         "jest/no-hooks": OFF,
         "jest/no-mocks-import": OFF,
         "jest/prefer-expect-assertions": OFF,
+        "@typescript-eslint/unbound-method": OFF,
+        "jest/unbound-method": OFF,
```

Comment:
It is frustrating that the only reason we have to turn this on is because of `Cookie.get` 😔